tests/baselines/reference/docker/vscode.log

```diff
@@ -1,24 +1,11 @@
-Exit Code: 1
+Exit Code: 0
 Standard output:
 yarn run vX.X.X
 $ gulp compile --max_old_space_size=4095
 [XX:XX:XX] Node flags detected: --max_old_space_size=4095
 [XX:XX:XX] Using gulpfile /vscode/gulpfile.js
-[XX:XX:XX] Error: /vscode/extensions/typescript-language-features/src/typescriptServiceClient.ts(738,37): 'command' is specified more than once, so this usage will be overwritten.
```

Comment:
this is -- I hope -- due to my pointing out this error to Matt.

Reply:
later: no, this is a bogus error, and the code hasn't changed: https://github.com/microsoft/vscode/blob/f0942786b4ee338f02449e2e5751091e028eb898/extensions/typescript-language-features/src/typescriptServiceClient.ts#L730 (although it's at line 730 instead of 738 now)

I don't think this was intentionally fixed, so I'm not sure what is going on.

Reply:
First change was #37126, Feb 28 5:42 PM.

Reply:
Oh, I wasn't looking at master on github. Here is the commit: https://github.com/microsoft/vscode/commit/d4dc9e09a2499da6396d2009625f5ca96369bf07#diff-e98bb4857e68818d83861d2e06773bd9